src/main/webapp/404static.xhtml

```diff
@@ -0,0 +1,110 @@
+<?xml version='1.0' encoding='UTF-8' ?>
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:h="http://java.sun.com/jsf/html" lang="en">
+    <head>
+        <title>Dataverse - Not Found</title>
```

Comment:
I'm seeing some hard coded English on this page. Should we be using something like..

```
#{bundle['error.404.page.title']}
```

here? And other bundle entries from the old 404.xhtml?

Reply:
Sure, why not. 
The words "copyright ..." etc. should be ok to stay hard-coded in English though.